draft-ietf-quic-transport.md

```diff
@@ -4107,47 +4111,48 @@ address of the client; see {{address-validation}}.
 
 The Path Maximum Transmission Unit (PMTU) is the maximum size of the entire IP
 packet including the IP header, UDP header, and UDP payload.  The UDP payload
-includes the QUIC packet header, protected payload, and any authentication
-fields.  The PMTU can depend on path characteristics, and can therefore change
-over time.  The largest UDP payload an endpoint sends at any given time is
-referred to as the endpoint's maximum packet size.
+includes QUIC packet headers, protected payload, and any authentication fields.
```

Comment:
I'm unclear if this is written with a single QUIC packet in mind or multiple.  Suggestion follows.

```suggestion
includes one or more QUIC packets.
```

Reply:
I changed "header" to "headers" to reflect that, but I can change it to "one or more QUIC packet headers, ...". I think it's worth spelling out the fields though.